README.md

```diff
@@ -1,3 +1,23 @@
+# What is louis-db ?
+
+Louis-db is the database of [Louis](https://github.com/ai-cfia/louis). It includes all of the python and bash scripts as well as sql functions. It uses ada-002 API (Application Programming Interface) for semantic search. 
+
+- The *bin* folder : This folder contains all of the bash scripts that would be useful to set up the database.
+
+- The *louis* folder : This folder contains python files that allows connections to the database or the api.
+
+- The *postgres* folder : This folder contains the bash script that will set up the docker container.
+
+- The *sql* folder : This folder holds all of the sql functions and scripts.
+
+- The *tests* folder : This is the test folder, it allows you to test the code.
+
+Here is the database schema :
+![database schema](database-schema.png)
```

Comment:
I feel like the schema is missing some tables

Reply:
I agree, however I'm not sure if it's because we add table when we do our code

Reply:
I don't remember adding tables